make/cpp2asm_support.cmake

```diff
@@ -132,7 +132,14 @@ endif ()
 ##################################################
 # Preprocessor location and flags
 
-if (UNIX)
+if (MSVC)
+  set(CMAKE_CPP ${CMAKE_C_COMPILER})
+
+  set(CPP_KEEP_COMMENTS /C)
+  set(CPP_NO_LINENUM /EP)
+  set(CPP_KEEP_WHITESPACE "")
+  set(CMAKE_CPP_FLAGS "/nologo")
+else()
```

Comment:
nit: missing space: please match the style of the existing code

Reply:
Fixed.